Core/COEditingContext.m

```diff
@@ -911,26 +964,9 @@ a correct deletion status (critical to update the cross
 		}
 		else if (![deletedPersistentRootUUIDs containsObject: persistentRootUUID])
 		{
-			/* For a finalized persistent root, insertedOrUndeleted is nil */
-			COPersistentRoot *insertedOrUndeleted = [self persistentRootForUUID: persistentRootUUID];
-
-			if (insertedOrUndeleted != nil)
```

Comment:
That branch of the if() only executes if the persistent root is not currently loaded, so I guess at that point any references to it must have already been fixed